packages/core/test/tooltip/tooltipTests.tsx

```diff
@@ -20,15 +20,65 @@ import * as React from "react";
 import { spy, stub } from "sinon";
 
 import { Classes } from "../../src/common";
+import * as Errors from "../../src/common/errors";
 import { Button, Overlay2 } from "../../src/components";
 import { Popover } from "../../src/components/popover/popover";
 import { Tooltip, type TooltipProps } from "../../src/components/tooltip/tooltip";
 
 const TARGET_SELECTOR = `.${Classes.POPOVER_TARGET}`;
 const TOOLTIP_SELECTOR = `.${Classes.TOOLTIP}`;
-const TEST_TARGET_ID = "test-target";
 
 describe("<Tooltip>", () => {
+    describe("validation", () => {
```

Comment:
Any reason to repeat these tests from popover? It feels like this is just testing the underlying popover component and so is the wrong place for these tests, which already exist on the underlying popover.

Reply:
See other comment. Basically we are modifying some of the passed values in `Tooltip`, so wanted to make sure `Popover` is receiving the same values and giving the same warnings.

Adding these tests helped to discover a bug with my original code changes; one was failing. Has since been fixed, but that shows how the tests benefit.

Reply:
Just introduced another failure to these tests with my last change, proved again that we need em!